contrib/gitian-descriptors/gitian-osx.yml

```diff
@@ -6,40 +6,37 @@ suites:
 architectures:
 - "amd64"
 packages:
+- "bsdmainutils"
 - "ca-certificates"
 - "curl"
-- "g++"
+- "clang"
+- "lld"
+- "llvm"
+- "llvm-dev"
```

Comment:
In `depends/README.md`, requires `g++`:

https://github.com/chaintope/tapyrus-core/blob/master/depends/README.md#for-macos-cross-compilation

Which one correct?

Reply:
It is the documentation that is not updated. I'll update it according the list in the CI. CI library list is the most recent and accurate.